client/app/views/templates/contacts/merge.jade

```diff
@@ -0,0 +1,89 @@
+extends ../../../lib/views/templates/base/dialog.jade
+
+
+block content
+    include helpers.jade
+
+    mixin radio(options, fieldName)
+      ul
+          - for (var i=0; i< options.length; i++)
+              -var option = options[i]
+                  li
+                      label
+                        if model[fieldName] == option.index
+                            input(type="radio", name=fieldName,
+                                value=option.index, checked)
+                        else
+                            input(type="radio", name=fieldName,
+                                value=option.index)
+
+                        i
+                        = option.value
+
+
+    h1= t('merge title')
+
+    dl
+        if model.mergeOptions.avatar != null
+          dt= t('merge choose avatar')
+          dd
+            ul.avatars
+                - for (var i=0; i< model.mergeOptions.avatar.length; i++)
+                    -var option = model.mergeOptions.avatar[i]
+                    li
+                        label
+                            if model.avatar == option.index
+                                input(type="radio", name="avatar",
+                                    value=option.index, checked)
+                            else
+                                input(type="radio", name="avatar",
+                                    value=option.index)
+
+                            img.avatar(src=option.value)
+
+
+        if model.mergeOptions.fn != null
+          dt= t('merge choose name')
+          dd
+            +radio(model.mergeOptions.fn, "fn")
+
```

Comment:
Can't we rely on jade mixin to build the selectors components? It can be more readable / maintanable.